test/validate/eia_test.py

```diff
@@ -40,18 +41,19 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_415_232, 1_415_232, 118_550),
+        ("bf_eia923", 1_415_328, 1_415_328, 118_558),
         ("bga_eia860", 129_869, 129_869, 129_869),
-        ("frc_eia923", 596_855, 244_403, 24_064),
-        ("gen_eia923", None, 5_171_930, 432_657),
+        ("boil_eia860", 74_146, 74_146, 74_146),
```

Comment:
Wow, that's a lot fewer than for the generators. I guess that's because there are a bunch of generators that don't burn anything and so don't have boilers.

Reply:
It seems roughly proportional to the bga table, but yes it is only applicable to this subset of the generators:

> For plants with a total steam-electric nameplate capacity of 10 MW or greater:
> Complete SCHEDULE 6, Part A for existing and planned boilers and associated equipment that serve
> combustible-fueled steam electric generator(s) and/or combined cycle steam generator(s) with duct
> firing.